libs/core/src/lib/shellbar/shellbar-actions/shellbar-actions.component.html

```diff
@@ -56,5 +54,6 @@
     </ng-container>
 </ng-container>
 
-
-<ng-content select="fd-product-switch"></ng-content>
+<div [ngClass]="{'fd-shellbar__action': productSwitchComponent}">
```

Comment:
Maybe consider using
```suggestion
<div class="fd-shellbar__action" *ngIf="productSwitchComponent">
```